src/data_ingestion/haida_text_processor.py

```diff
@@ -0,0 +1,21 @@
+def convert_optima_haidatron_to_standard_orthography(optima_haidatron_text):
+    replacements = {
+        "Ñ": "K̲",
+        "ñ": "k̲",
+        "Ý": "X̲",
+        "ý": "x̲",
+        "Ç": "G̲",
+        "ç": "ɢ̲",
+        "Þ": "X̂",
+        "þ": "x̂",
+        "Ð": "Ĝ",
+        "ð": "ɢ̂"
+        }
+    
+    output_chars = []
+
+    for c in optima_haidatron_text:
+        # if the char is not in the replacement list, we append the char without replacement
+        output_chars.append(replacements.get(c,c))
```

Comment:
So `.get()` will just return `c` of `optima_haidatron_text` if it isn't a key in `replacements` and otherwise it will return the value of `c` from `replacements`?